chain/network/src/peer_manager/network_state/mod.rs

```diff
@@ -358,29 +399,44 @@ impl NetworkState {
                 return false;
             }
         }
-        match self.graph.routing_table.find_route(&clock, &msg.target) {
-            Ok(peer_id) => {
-                // Remember if we expect a response for this message.
-                if msg.author == my_peer_id && msg.expect_response() {
-                    tracing::trace!(target: "network", ?msg, "initiate route back");
-                    self.graph.routing_table.add_route_back(&clock, msg.hash(), my_peer_id);
-                }
-                return self.tier2.send_message(peer_id, Arc::new(PeerMessage::Routed(msg)));
+        match tier {
+            tcp::Tier::T1 => {
+                tracing::debug!(target:"test", "sending msg over TIER1");
+                let peer_id = match &msg.target {
+                    PeerIdOrHash::Hash(hash) => {
```

Comment:
add a comment - that if the message is a response - we try to load the target from the route back cache.

Reply:
done